town-hall/2021-11-05.md

```diff
@@ -0,0 +1,130 @@
+- Newsletter
+    - We are working on a #newsletter
+    - Thank you `camo` and `Kairon`
+    - Retrospective newsletter will go out soon, please sign up!
+    - Looking for a sponsorship to be first on-chain revenue generator
+    - [sign up here](https://developerdao.substack.com/)
+
+## IRL + Live Events
+
+- Use the #IRL channel to network in your locality
+- DevDAO at [CascadiaJS](https://twitter.com/MichelleBakels/status/1456084953511317508/photo/1)
+- If your region is not listed, use the #request-new-region channel to add
+
+## Season 0 Proposal
+
+- Season 0 proposal is out [on Discourse](https://forum.developerdao.com/t/p3-developer-dao-season-0/219/18)
+- Assuming vote is in favor on Sunday, it will be moved to a [Snapshot vote](https://snapshot.org/#/devdao.eth)
+- Engagement was around 1/4, other DAOs are usually just a few percent
+- [Boardroom](https://www.boardroom.info/) may be used in the future to centralize our decentralization
+- Three Guilds to be formed in coming weeks
+    - Community Guild
+    - Development Guild
+    - Design/Marketing/BizDev Guild
+- Announcement post on Mirror next weekend
+    - after 5pm UTC next Friday - keep in mind daylight savings times
+- We are looking to build an inclusive community, message `nader` if you know anyone who needs help with minting or have any ideas
+    - Roadmap for different guilds could help break down barriers for anyone new to the development world
+    - Potentially could have Docents to help welcome people to the community
+- [If you want to go fast go alone, if you want to go far go together...](https://forum.developerdao.com/t/if-you-want-to-go-fast-go-alone-if-you-want-to-go-far-go-together-if-you-want-to-go-nowhere-start-a-dao/122?u=nathanng.eth)
+- We're looking into options for season tokens on L2s/Solana
+
+## GitCoin Partnership
+
+- [Gitcoin](https://gitcoin.co) was one of the first public resources to learn and earn on-chain revenue by building in the Web3 space
+    - Also are funding projects in the Web3 space
+    - There is potential for Gitcoin to dedicate funds to our DAO
+    - Enough to fund our treasury for a while and compensate contribution
+    - We will do an announcement when this is finalized
+- One of their projects is the [MoonshotCollective](https://moonshotcollective.space/)
+- 2-3 weeks on an optimistic time-scale
+
+# Notes
+
+- Grants
+    - $200MM grant with Avalanche
+        - we can apply as a DAO
+    - $1BB Algorand just [announced](https://www.youtube.com/watch?v=DSDCFP903m8)
+- [Kernel](http://kernel.community/en/) has a learning track path where you'll lock up 400 DAI
+- Are we considering Polygon for the pixel avatars/future tokens?
+    - We can do a vote on this!
+- Now is a good time for us to begin to flesh out guidelines, reporting mechanisms, etc.
+    - we could look into to have cohort-style onboarding in future
+    - Learning path tokens
```

Comment:
```suggestion
    - Learning path tokens
- Q&A at the end
  - Discussed the onboarding is currently very confusion and needs to be improved
    - Key gaol for Community Guild during Season 0
  - Discussed the whilst we speak about inclusivity and diversity, this is currently lacking in the Core Team who are on stage speaking for the DAO.
    - Suggested we open up a space for discussing this further to get feedback from everyone in the community on how we can improve here.
- Discussed the exclusive nature of having to pay gas fees for the NFT
  - Acknowledge that this isn't ideal. Need someone way to gate membership, when launched gas fee's were not as high as they are now so wasn't as big a barrier. Doesn't change the fact that some people wont be able to afford, was best option at the time.
  - Some effort has been made by `Nader` and others to other membership to people who otherwise cannot afford their won via give aways but we could do a lot.
  - In the future when we could consider sponsorships, bounties, more NFTs on layer 2's for access.
```